lib/skate_web/views/error_helpers.ex

```diff
@@ -4,4 +4,17 @@ defmodule SkateWeb.ErrorHelpers do
   """
 
   use Phoenix.HTML
+
+  @doc """
+  Generates a tag for inlined form input errors.
+  """
+  def error_tag(form, field) do
+    case form.errors[field] do
+      {error, _} ->
+        content_tag(:span, error)
```

Comment:
This has basically no styling right now, so it doesn't look particularly nice. I'm on the fence about whether to change that, because:
1. It's an internal tool that's used basically only by us.
2. The exact way to give it nice styles will change with HEEx.
3. If we do declarative test groups (which I think is a good idea), then this error state will go away.